lambdas/db-migration/webpack.config.js

```diff
@@ -39,7 +39,26 @@ module.exports = {
           },
         ],
       },
+      // NOTE: This is dark magic that prevents Knex from failing when
```

Comment:
Please add a warning in the README for this lambda re: this, as an update to knex could cause all manner of grief.

Reply:
14f5a2fb628a950e221d1116c3f2e861f9fbf9db